From c5128f9bafaa37a8e63989032372a5ff36dc82c4 Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Sat, 27 Sep 2014 19:43:08 +0800
Subject: [PATCH] MLK-9637-1 usb: test: define otg_srp_reqd wIndex byte value

This patch adds otg_srp_reqd wIndex high byte value for otg test mode.

Acked-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 34660f7aa14972630622406b9dbfc7a106d8d15d)
---
 include/uapi/linux/usb/ch9.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/usb/ch9.h b/include/uapi/linux/usb/ch9.h
index c1856f3..5a5cada 100644
--- a/include/uapi/linux/usb/ch9.h
+++ b/include/uapi/linux/usb/ch9.h
@@ -135,6 +135,7 @@
 #define	TEST_SE0_NAK	3
 #define	TEST_PACKET	4
 #define	TEST_FORCE_EN	5
+#define	TEST_OTG_SRP_REQD	6
 
 /*
  * New Feature Selectors as added by USB 3.0
-- 
1.8.0

